cli/loader_lib.c

```diff
@@ -133,7 +136,10 @@ JL_DLLEXPORT const char * jl_get_libdir()
     Dl_info info;
     if (!dladdr(&jl_get_libdir, &info)) {
         jl_loader_print_stderr("ERROR: Unable to dladdr(&jl_get_libdir)!\n");
-        jl_loader_print_stderr3("Message:", dlerror(), "\n");
+        char *dlerr = dlerror();
+        if (dlerr != NULL) {
+            jl_loader_print_stderr3("Message:", dlerr, "\n");
+        }
```

Comment:
I'm actually not sure this `dlerror` is ever useful: https://man7.org/linux/man-pages/man3/dladdr.3.html
```
       If the address specified in addr could not be matched to a shared
       object, then these functions return 0.  In this case, an error
       message is not available via [dlerror(3)](https://man7.org/linux/man-pages/man3/dlerror.3.html).
```